riotfile.py

```diff
@@ -2595,13 +2595,11 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         Venv(
             name="gunicorn",
             command="pytest {cmdargs} tests/contrib/gunicorn",
-            pkgs={"requests": latest},
+            pkgs={"requests": latest, "gevent": latest},
+            env={
+                "AGENT_VERSION": "latest",
+            },
```

Comment:
```suggestion
```

This environment variable is  used in the following test suite: [integration](https://github.com/DataDog/dd-trace-py/blob/1.x/tests/integration/test_integration.py#L25). It's not used in gunicorn tests